modules/veteran/app/controllers/veteran/v0/base_accredited_representatives_controller.rb

```diff
@@ -14,48 +14,24 @@ class AccreditedRepresentativesController < ApplicationController
       DEFAULT_PAGE = 1
       DEFAULT_PER_PAGE = 10
       DEFAULT_SORT = 'distance_asc'
-
-      PERMITTED_TYPES = 'attorney'
-
       PERMITTED_REPRESENTATIVE_SORTS = %w[distance_asc first_name_asc first_name_desc last_name_asc
                                           last_name_desc].freeze
+
       def index
-        collection = Common::Collection.new(model_klass, data: accreditation_query)
+        collection = Common::Collection.new(Veteran::Service::Representative, data: representative_query)
         resource = collection.paginate(**pagination_params)
 
         render json: resource.data,
                serializer: CollectionSerializer,
-               each_serializer: serializer_klass,
+               each_serializer: serializer_class,
```

Comment:
similar comment about relying on the sub classes to implement this method

Reply:
I added #serializer_class to BaseAccreditedRepresentativesController which raises NotImplementedError.